renga_deployer/utils.py

```diff
@@ -81,3 +82,22 @@ def validate_uuid(s, version=4):
         return False
 
     return s == str(uid)
+
+
+def validate_uuid_args(*names):
+    """Check that input arguments are valid UUIDs."""
+    def decorator(func):
+        @wraps(func)
+        def wrapper(*args, **kwargs):
+            """Check the input arguments and return."""
+            if not all([name in kwargs for name in names]):
```

Comment:
You should not do it runtime but at import time using `inspect` module: https://docs.python.org/3/library/inspect.html#inspect.signature